lib/rules/no-test-import-export.js

```diff
@@ -45,7 +46,10 @@ module.exports = {
       ImportDeclaration(node) {
         const importSource = node.source.value;
 
-        if (importSource.endsWith('-test') && !isTestHelperImportSource(importSource)) {
+        if (
+          importSource.endsWith('-test') &&
+          !isTestHelperImportSource(resolve(dirname(context.getFilename()), importSource))
```

Comment:
Ember makes some of the import paths difficult to resolve, since it does things like drop the `/addon` or `/app`. I'm not sure you're able to import things from the dummy app in your tests, but if you can this might not resolve correctly.

Reply:
That's true. I'm not sure how we could handle that, but fortunately it should be very unlikely for that to affect us with test helper imports.